tests/test_categories.py

```diff
@@ -0,0 +1,161 @@
+import os
+import json
+import pytest
+from app import app
+from apistar.test import TestClient
+from freezegun import freeze_time
+from .test_api import load_poi
+
+BBOX_PARIS="2.252876,48.819862,2.395707,48.891132"
+
+def load_poi(file_name, mimir_client):
+    """
+    Load a json file in the elasticsearch and returns the corresponding POI id
+    """
+    filepath = os.path.join(os.path.dirname(__file__), 'fixtures', file_name)
+    with open(filepath, "r") as f:
+        poi = json.load(f)
+        poi_id = poi['id']
+        mimir_client.index(index='munin_poi',
+                        body=poi,
+                        doc_type='poi',
+                        id=poi_id,
+                        refresh=True)
+
+@pytest.fixture(autouse=True)
+def load_all(mimir_client, init_indices):
+    """
+    fill elasticsearch with 4 POI:
```

Comment:
these fixtures should be updated to take into account of the fafnir last changes https://github.com/QwantResearch/fafnir/pull/25